v3/src/components/graph/adornments/univariate-measures/standard-error/standard-error-adornment-component.tsx

```diff
@@ -83,14 +80,12 @@ export const StandardErrorAdornmentComponent = observer(
       if (!numericAttrId || !dataConfig) return
       const labelSelection = select(labelRef.current)
       const labelCoords = measure.labelCoords
-      const lineHeight = 20
-      const topOffset = lineHeight * (adornmentsStore?.getLabelLinesAboveAdornment(model, isGaussianFit) ?? 0)
```

Comment:
As with the normal curve adornment, if the user has not manually moved the measure label, getLabelLinesAboveAdornment is necessary to prevent overlap with other adornments.